TRANSLATING.md

```diff
@@ -0,0 +1,142 @@
+# Translate Script Lab to other languages
+
+This document describes how to create a new translation for Script Lab or improve an existing translation.
+
+## Topics
+* [Prerequisites](TRANSLATING.md#prerequisites)
+* [Understanding the Script Lab structure for translating](TRANSLATING.md#structure)
+* [Create a new translation](TRANSLATING.md/#create)
+* [Improve an existing translation](TRANSLATING.md#improve)
+* [Testing your translation](TRANSLATING.md#testing)
+* [Script Lab languages](TRANSLATING.md/#translations)
+
+<a id="prerequisites"></a>
+## Prerequisites
+
+Fork this project into your GitHub account and use branches to update existing files or add new files. When you are ready, send us a pull request. Please note, that if you are not employed by Microsoft and you have not already signed the Microsoft Contribution License Agreement, you will be asked to sign the agreement before your pull request is accepted.
+
+We recommend you to download and install Visual Studio Code from here <https://code.visualstudio.com/> and follow the instructions from here [CONTRIBUTING.md](CONTRIBUTING.md) how to run the playground from source and having a local copy of the project. 
+
+<a id="structure"></a>
+## Understanding the Script Lab structure for translating
+
+The Script Lab code is mainly composed of two parts: the server part and the client part. Each part has its own files to be modified and/or added for creating or improving a translation. Additionally, the project includes 4 manifest files which also include translatable strings.
+
+Basically, the structure is represented by the following table. For the server and client part, each translation is stored in separate files named to the corresponding language. For each manifest file, the translations are included in different sections of the file.
```

Comment:
I think this table would be much easier to understand if it was actually three separate tables: server, client, and manifest.